applications/ShapeOptimizationApplication/custom_utilities/damping/damping_utilities.h

```diff
@@ -94,10 +94,10 @@ class DampingUtilities
           mMaxNeighborNodes( DampingSettings["max_neighbor_nodes"].GetInt() )
     {
         BuiltinTimer timer;
-        std::cout << "\n> Creating search tree to perform damping..." << std::endl;
+        KRATOS_INFO("") << "\n> Creating search tree to perform damping..." << std::endl;
```

Comment:
why don't you add a label? Makes it easier to track where the messages come from

Reply:
Idem

Reply:
@dbaumgaertner what label would you prefer?

```
> Creating search tree to perform damping...
```
becomes
```
ShapeOpt: Creating search tree to perform damping...
```
or 
```
KSO: Creating search tree to perform damping...
```

Reply:
i would for `ShapeOptimization` or `ShapeOpt`

Reply:
```
ShapeOptimizationApplication: Creating search tree to perform damping...
```
is to long